### ISSUE TITLE
Progress bar and actual progress are completely out of sync

### ISSUE THREAD

**Issue by PD75** [kalkih/mini-media-player#409]
media_duration: 549.266 (= 09:09)
media_position: 298.78 (=04:59)

![image](https://user-images.githubusercontent.com/10294795/92205596-522e0f00-ee86-11ea-8b7e-4d373e124333.png)

**Comment by kalkih** [comment]
You need to provide more information so I can try to reproduce this on my end.
When does it happen? Does it always display out of sync? What integrations is the media player? What's your card configuration

**Comment by PD75** [comment]
```
  - entity: media_player.spotify_
    group: true
    hide:
      icon_state: false
      power: true
      progress: false
      runtime: false
      shuffle: false
      volume: true
    info: short
    sound_mode: full
    source: full
    state_filter:
      - operator: '!='
        value: unavailable
    type: 'custom:mini-media-player'
```

 ```
- entity: media_player.music_player_daemon
    group: true
    hide:
      power_state: false
      progress: false
      runtime: false
      shuffle: false
      volume: true
    icon: 'mdi:music-circle'
    info: short
    source: icon
    state_filter:
      - operator: '!='
        value: unavailable
    type: 'custom:mini-media-player'
```

more info to come

**Comment by PD75** [comment]
media_duration: '497'
media_position: '100:497'

![image](https://user-images.githubusercontent.com/10294795/92691626-1e9a2b80-f343-11ea-8065-0c0e4ab07af7.png)

**Comment by PD75** [comment]
I work and check ones in a while, the NaN on MPD is always. It comes out of sync over time, not sure what I do. I have the browser open on another computer I RDP to. What other data do you need?

**Comment by lbartk** [comment]
Hi, 
I have experienced exactly the same issue. 

> volume_level: 0.08
is_volume_muted: false
media_content_id: 'spotify:track:6QgjcU0zLnzq5OrUoSZ3OK'
media_content_type: music
media_duration: '163'
media_position: '13:163'
media_position_updated_at: '2020-09-16T13:32:00.384990+00:00'
media_title: Feel It Still
media_artist: Portugal. The Man
media_album_name: Woodstock
shuffle: false
friendly_name: MPD
supported_features: 65471

![image](https://user-images.githubusercontent.com/52855447/93344427-fa9e8480-f831-11ea-8fa1-7ab1bb9beece.png)

It looks like it could not read properly the "media_position".

**Comment by lbartk** [comment]
I had to reinstall the whole server today and after reinstall works perfectly.

**Comment by kalkih** [comment]
Yes, this was fixed in the latest Home Assistant version 0.115.0, see https://github.com/home-assistant/core/pull/39390
